defaults/main.yml

```diff
@@ -72,6 +72,7 @@ nexus_public_hostname: 'nexus.vm'
 nexus_public_scheme: https
 
 # How should the role access the API for provisionning
+nexus_rest_api_endpoint_base: "service/rest"
```

Comment:
This name is conflicting/confusing. There is a version based detection of the rest endpoint in `nexus_install.yml` which sets `nexus_rest_api_endpoint` used in `call_script.yml, `declare_script_each.yml` and `nexus_install.yml`.

Unfortunately, at this point, the var is pointing directly to the `script` endpoint (which is historically the only one used).

To clean-up the role's code, I'm considering dropping support for nexus version < 3.11 which would remove this detection (along wiht some other path's detection) and would let us refactor the existing files to use a base as above for all api interactions.